### DISCUSSION TITLE
Mechanism to detect clicks on plot area and axis separately?

### DISCUSSION THREAD

**Discussion by bryaan** [JuliaPlots/Makie.jl#4260] [Q&A]
How can I detect clicks on only the xaxis?  How about only on the plot area where series go?

I have tried:

`is_mouseinside(ax.scene)`

But that detects either axis and also triggers for clicks inside the plot area.
I need the xaxis and plot area separately.

**Comment by ffreyer** [comment]
`is_mouseinside` just checks if the mouse is within `scene.viewport`. Other than checking for areas you can use `pick()` to check for plots, but that's it
